packages/components/src/components/app-card/app-card.jsx

```diff
@@ -0,0 +1,147 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { isMobile } from '@deriv/shared';
+import AppCardHeader from './app-card-items/app-card-header.jsx';
+import AppCardBody from './app-card-items/app-card-body.jsx';
+import AppCardActions from './app-card-items/app-card-actions.jsx';
+import AppCardFooter from './app-card-items/app-card-footer.jsx';
+import { useHover } from './hooks/use-hover';
+
+const RealAppCardBAckground = ({ is_swap_free, variant }) => (
+    <svg className='dc-app-card__wrapper--real-background'>
+        <path
+            d={
+                variant === 'default'
+                    ? isMobile()
+                        ? 'M0 0h272v12L0 44V0z'
+                        : 'M0 0h280v16L0 60V0z'
+                    : 'M0 0h216v9L0 32V0z'
+            }
+            fill={is_swap_free ? '#D8E4E6' : '#F0F0F0'}
+        />
+        <path
+            d={
+                variant === 'default'
+                    ? isMobile()
+                        ? 'M0 0h272v12L0 30V0z'
+                        : 'M0 0h280v16L0 40V0z'
+                    : 'M0 0h216v9L0 21V0z'
+            }
+            fill={is_swap_free ? '#BDD2D5' : '#F9F9F9'}
+        />
+    </svg>
+);
+
+const AppCard = ({
+    amount,
+    app_icon,
+    app_name,
+    broker,
+    currency,
+    getCardLabels,
+    is_swap_free,
+    is_virtual,
+    linked_wallet,
+    login_id,
+    onAddRealClick,
+    onDepositClick,
+    onPlayClick,
+    onSettingsClick,
+    onTransactionsClick,
+    onWithdrawClick,
+    server,
+    show_footer,
+    show_hover_actions,
+    variant = 'default',
+}) => {
+    const [card_ref, is_hovered] = useHover();
+    const getFontColor = () => {
+        if (is_virtual) return 'colored-background';
+        return 'general';
+    };
+    return (
+        <React.Fragment>
+            <div
+                className={classNames('dc-app-card__wrapper', {
+                    'dc-app-card__wrapper--virtual': is_virtual,
+                    'dc-app-card__wrapper--real': !is_virtual,
+                    [`dc-app-card__wrapper--${variant}`]: variant,
```

Comment:
To make it easier for other developers to figure out which classes could exist I'd suggest writing out all the variants rather than using a variable class name. e.g.:

```JS
'dc-app-card__wrapper--default': variant === 'default',
'dc-app-card__wrapper--mini': variant === 'mini',
'dc-app-card__wrapper--micro': variant === 'micro',
```